MountSionCBSSecondary/SchoolWebsite
# 1st6thbooklistnavbar

# Description

Adjusted Navigation Bar on website to have adjusted booklists

## What

Pages for Junior Cycle and Senior Cycle booklists have been separated into different classes.

## Why

Parents can access booklists easier

## Verification Steps

http-server, navigate to Booklists and make sure all content is added to all booklists.


# Checklist:
